plugins/rest/Dockerfile

```diff
@@ -1,29 +1,20 @@
-FROM rapid7/insightconnect-python-3-38-slim-plugin:4
-# The three supported python parent images are:
-# - komand/python-2-plugin
-# - komand/python-3-plugin
-# - komand/python-pypy3-plugin
-#
-LABEL organization=komand
+FROM rapid7/insightconnect-python-3-38-plugin:4
 
-# Add any custom package dependencies here
-# NOTE: Add pip packages to requirements.txt
+LABEL organization=rapid7
+LABEL sdk=python
 
-# End package dependencies
-
-# Add source code
 WORKDIR /python/src
+
 ADD ./plugin.spec.yaml /plugin.spec.yaml
-ADD . /python/src
+ADD ./requirements.txt /python/src/requirements.txt
 
-# Install pip dependencies
 RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
-# Install plugin
-RUN python setup.py build && python setup.py install
+ADD . /python/src
+
+RUN	python setup.py build && python setup.py install
 
-# needed for SSL Verify
-USER root
+# User to run plugin code. The two supported users are: root, nobody
+USER nobody
```

Comment:
just double checking that changing to nobody wont break anything.
It looks like there was a comment here previously saying it was necessary for SSL Verify.

Reply:
Try to add `sdk` field to `plugin.spec.yaml` file. That would affect the `Dockerfile` 😄 . For example:
```
sdk:
  type: fyll
  version: 5
  user: root
  packages:
    - curl
    - another_package
    - etc...
```